src/drivers/mct/main/prep_ocn_mod.F90

```diff
@@ -48,6 +48,10 @@ module prep_ocn_mod
 
   public :: prep_ocn_get_x2oacc_ox
   public :: prep_ocn_get_x2oacc_ox_cnt
+#ifdef SUMMITDEV_PGI
```

Comment:
OK, I added the preprocessor restriction.

Reply:
Also need to add this to references in `cime/src/drivers/mct/main/seq_rest_mod.F90`.

Reply:
@amametjanov I don't see any references to "dummy" or "pgi" in that file.

Reply:
Oh, the latest cime merge took out those `dummy_pgibugfix = associated(x2oacc_ox)` workarounds from that PR. @sarats may want to re-check with the latest E3SM master.

Reply:
@amametjanov you're right, it was lost in the recent ESMCI integration, I will put it back.

Reply:
@amametjanov @sarats , to avoid this in the future, it might be best to make driver changes under ESMCI.

Reply:
@jgfouca Understood, thanks for fixing.